Dumpers/python/Cfg_RecoSimDumper_cfi.py

```diff
@@ -38,6 +38,7 @@
     saveSimhits                       = cms.bool(True),  #save simHits information
     saveRechits                       = cms.bool(True),  #save recHits information
```

Comment:
Minor comment/question

Do you actually think that some options could be turned to False (such as for instance saveSimHits and saveRechits), for the sake of having slimmer dumped_files? 

Or do those flags need to be set to true in order to compute higher-order variables (PFClusters et co.) ?

Reply:
mmm, my naive answer would be: if we kept them for all this time there was a reason. 
However, if I go back to the code, saveSimhits we can definitely set to false. 
saveRecHits needs to be true to calculate the shower shape variables. but the logic is flawed. I will push a commit to fix it

Reply:
Thanks a lot!